web/src/buddy_data.ts

```diff
@@ -393,12 +397,35 @@ function get_filtered_user_id_list(user_filter_text: string): number[] {
                 base_user_id_list = [...base_user_id_set];
             }
         }
+
+        // We always want to show conversation participants even if they're inactive.
+        const base_user_id_set = new Set([
+            ...base_user_id_list,
+            ...get_conversation_participants(),
```

Comment:
`get_conversation_participants` is not necessarily super cheap, we should aim to avoid calling it multiple times in a single code path.

Reply:
Fair, I'll pass it to those two functions here.

Reply:
It'll get called twice, once here and once when building the buddy list, but I think it's a decent amount more complicated to avoid that.